src/utils/perfherder/chartJs/perfherderFormatter.js

```diff
@@ -17,28 +17,29 @@ const generateInitialOptions = series => {
     tooltips: {
       callbacks: {
         footer: (tooltipItems, data) => {
-          const tooltipData = [];
-          let delta = 'n/a';
-          let deltaPercentage = 'n/a';
-          let dataset = 'n/a';
-          let currentData = 'n/a';
+          // get data from all points of selected series
+          const dataset = data.datasets[tooltipItems[0].datasetIndex].data;
+          // get data from selected point
+          const currentData = dataset[tooltipItems[0].index].y;
+          const tooltipData = []; // footer's text lines will be stored here
+          let delta = 0;
+          let deltaPercentage = 0;
 
           if (tooltipItems[0].index > 0) {
-            dataset = data.datasets[tooltipItems[0].datasetIndex].data;
-
-            currentData = dataset[tooltipItems[0].index].y;
             const previousData = dataset[tooltipItems[0].index - 1].y;
 
-            delta = (currentData - previousData).toFixed(2);
-            deltaPercentage = (
-              ((currentData - previousData) / previousData) *
-              100
-            ).toFixed(2);
+            delta = currentData - previousData;
+            // [(c - p) / p] * 100 is equivalent to (c / p - 1) * 100
+            deltaPercentage = (currentData / previousData - 1) * 100;
           }
 
+          delta = delta.toFixed(2);
+          deltaPercentage = deltaPercentage.toFixed(2);
```

Comment:
@2alin it is good to chain .toFixed(2) on line 31 and 33 it will reduce line of code and make it look more clean. 

You can do 
`delta = (currentData - previousData).toFixed(2);`
`deltaPercentage = ((currentData / previousData - 1) * 100).toFixed(2);`

Reply:
Also, you can remove the comment. :)

Reply:
I moved the 'fixes' outside the conditional for a reason: to make the number format uniform even for the initial points which are not considered in the conditional. If I move it inside, as you suggest, I'll end up with Δ 0 (0%) for the initial point in comparison with all the other values like Δ 2.00 (12.13%) and even values that are not initial but end up being  Δ 0.00 (0.00%). In my opinion, it doesn't look good to have different number format for the same kind of values.

Reply:
This makes sense. Let's wait for @sarah-clements  to come and review then we should be able to merge this PR.

Reply:
If the value is 0, I think it's fine to have the format be `Δ 0 (0%)`. I don't think it is necessarily better to show `Δ 0.00 (0.00%)` since the decimals are irrelevant.